### DISCUSSION TITLE
Running Laravel Horizon on laragon

### DISCUSSION THREAD

**Discussion by berto309** [leokhoa/laragon#378] [Q&A]
Hello please how can  run horizon on laragon. After running php artisan horizon command, I had an error that I don't have pcntl ext.

**Comment by Pen-y-Fan** [comment]
See SO: <https://stackoverflow.com/questions/48577465/how-can-i-solve-laravel-horizon-v1-1-0-requires-ext-pcntl-the-requested-ph>

The **pcntl** extension isn't available on Windows.
